website/source/docs/builders/digitalocean.html.markdown

```diff
@@ -38,6 +38,9 @@ builder.
     can also be specified via environment variable `DIGITALOCEAN_API_TOKEN`,
     if set.
 
+-   `api_url` (string) - Non standard api endpoint url. Usefult in case of
+    DigitalOcean compatibility API hosting.
+
```

Comment:
I suggest:
```markdown
-   `api_url` (string) - Non standard api endpoint URL. Set this if you are
    using a DigitalOcean API compatible service. It can also be specified via
    environment variable `DIGITALOCEAN_API_URL`.
```
And it should be under optional since this have a default value.